persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_18/main.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/io;
```

Comment:
License headers?

Reply:
check and add missing licenses in other places

Reply:
Updated in [c7709b2](https://github.com/ballerina-platform/persist-tools/pull/48/commits/c7709b28a97d8f91b708b4bd3d483e0a0eebb499)